website/src/components/views/CreateJob.jsx

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import queryString from 'query-string'
+import { NavBar } from '../NavBar';
+
+class CreateJob extends React.Component {
+    constructor(props) {
+        super(props);
+        this.onSubmitWithProps = this.onSubmitWithProps.bind(this)
+
+        const queryValues = queryString.parse(props.location.search)
+        let cronJob = {}
+        props.cronJobs.map(item => {
+            if (queryValues.namespace === item.namespace && queryValues.cronjob === item.name) {
+                cronJob = item
+            }
+            return null
+        })
+        let formValues = {}
+        cronJob.config.options.map((option, index) => {
+            formValues[index] = option.default
+            return null
+        })
```

Comment:
Use `map` when you want to produce a new collection; not when you want to mutate an existing collection.
Using `reduce` would be better than `map` in this instance i.e.

```jsx
const formValues = cronJob.config.options.reduce((store, option, index) => {
    store[index] = option.default
    return store
})
```